views/AppView.js

```diff
@@ -0,0 +1,68 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .search-button': 'updateCurrentQuery',
+    'keypress #search-bar': 'enterHandler'
+  },
+
+  initialize(){
+    this.vidCollection = this.model.get('videos');
+
+    this.listenTo(this.vidCollection, 'reset', this.renderPlayer);
+    this.listenTo(this.vidCollection, 'reset', this.renderThumbnail);
+    this.listenTo(this.vidCollection, 'reset', this.assignMainPlayer);
+  },
+
+  updateCurrentQuery() {
+    if (this.$('#search-bar').val() != ''){
+      var query = this.$('#search-bar').val();
+      this.model.changeCurrentQuery(query);
+      this.$('#search-bar').val('');
+    }
+    
+  },
+
+  enterHandler(e) {
+    if (e.which === 13) {this.updateCurrentQuery()}
+  },
+
+  //I was torn on whether to keep renderThumbnail() and renderPlayer() separate or to combine them, it looked odd both ways. Let me know what you think!
+
```

Comment:
Style thing - no need for the new line below comments.

Also comments like this wouldn't normally go right in the code.

Btw, I think having them separate is good. Keep functions small and intentional, with one job.